docs/contribution_handling.md

```diff
@@ -0,0 +1,53 @@
+# WinUI Repo Contribution Handling
+
+The WinUI repo is intended both as a place for the WinUI community to discuss development with each other, and as a place to provide insight into upcoming changes and bug fixes the WinUI team is working on before updates are released. Here, we’ll outline how we handle feature requests and bugs that the community opens.
```

Comment:
A github repo shouldn't be for community discussion with each other - it should be for providing feedback to and discussions with the repo owners. Community discussions belong in the various developer forums.
Providing insights into upcoming changes should be in the form of blog posts, documentation and design PRs.

Reply:
Thanks @dotMorten, very astute. This was probably poorly phrased and was honestly just meant as a preamble to the rest of the document, so I didn't realize how it came across. I think you're right and I've updated the wording to be a little simpler and hopefully more accurate.